vue/sbc-common-components/src/components/BrowserVersionAlert.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <sbc-system-banner
+    v-if="browserSupported === false"
+    v-bind:show="browserSupported === false"
```

Comment:
You don't need both v-if and v-show.
(v-if conditionally renders the component)
(v-show conditionally set the display/none attribute, so the component is there but not visible, which may affect screen readers)
Also, a shortcut to "v-bind:show=" is simply ":show=". (Also line 6 below.)

Reply:
Changed